core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -568,36 +572,44 @@ private void addTableMacro(Connection connection, Method method) throws SQLExcep
     checkTableFunctionInModel(Smalls.TestStaticTableFunction.class);
   }
 
-  private CalciteAssert.AssertThat assertWithMacro(Class clazz) {
+  private CalciteAssert.AssertThat assertOverloadedWithMacro(Class<?>... clazz) {
+    String prefix = ""
```

Comment:
I think we should keep the old name. The passed in param is not necessary to be an array, so it maybe not overloaded.

Reply:
Ok, thanks for pointing this, reverted changing method name.